src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackageValidation.props

```diff
@@ -2,6 +2,6 @@
 <Project>
   <PropertyGroup>
     <DotNetPackageValidationAssembly Condition="'$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)..\tools\net472\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
-    <DotNetPackageValidationAssembly Condition="'$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)..\tools\netcoreapp3.1\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
+    <DotNetPackageValidationAssembly Condition="'$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)..\tools\net6.0\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
```

Comment:
Are we going to support overriding these tools via a NuGet package? Are you planning on adding such support on a separate PR?

Reply:
> Are we going to support overriding these tools via a NuGet package? Are you planning on adding such support on a separate PR?

yes the support is already there, ```Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets ``` is overriding this property in case of package reference.

Reply:
But you moved these files into the SDK, I don't see how these are being packed anymore.

Reply:
When using the package you don't just want to consume newer assemblies but also updates to the msbuild files which isn't possible with the current design.

Reply:
Yes.. these files should be included in the package and should either override the targets or set a property on the props file that causes the SDK to not longer include the inbox targets if ordering is an issue.

Reply:
@Anipik if you need inspiration on how to accomplish that, look at the ILLink.Tasks integration:
- https://github.com/mono/linker/blob/main/src/ILLink.Tasks/sdk/Sdk.props
- https://github.com/mono/linker/blob/b888d672e11588e2bbca1cd3eeaee30d53416897/src/ILLink.Tasks/build/Microsoft.NET.ILLink.Tasks.props#L15
- https://github.com/dotnet/sdk/blob/3983460336101a26a2304a5ccfc34ba379a015f6/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.props#L14